File: app/src/main/java/com/iuicity/xinjr/network/bean/resp/LoginResp.java
package com.iuicity.xinjr.network.bean.resp;

/**
 * @author: luosheng
 * @time: 2017/3/28 0028 12:44
 * @desc: <类的描述>
 */

public class LoginResp {


    /**
     * token : c200cb551e1aaa9b51898a09c6c226db
     * uid : 42
     * type : 1
     * status : 0
     */

    private String token;
    private String uid;
    private String type;
    private String status;

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: app/src/main/java/com/iuicity/xinjr/network/bean/resp/LoginResp.kt
package com.iuicity.xinjr.network.bean.resp

import android.os.Parcel
import android.os.Parcelable

/**
 * Created by Shui on 2017/7/7.
 */
data class LoginResp(val token: String, val uid: String, val type: String, val status: String) : Parcelable {
    constructor(parcel: Parcel) : this(
            parcel.readString(),
            parcel.readString(),
            parcel.readString(),
            parcel.readString())

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(token)
        parcel.writeString(uid)
        parcel.writeString(type)
        parcel.writeString(status)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<LoginResp> {
        override fun createFromParcel(parcel: Parcel): LoginResp {
            return LoginResp(parcel)
        }

        override fun newArray(size: Int): Array<LoginResp?> {
            return arrayOfNulls(size)
        }
    }
}